spec/factories/legal_framework_serialized_merits_task_list.rb

```diff
@@ -8,7 +8,10 @@
         application: {
           tasks: {
             latest_incident_details: [],
-            opponent_details: [],
+            opponent_name: [],
+            opponent_mental_capacity: [],
+            domestic_abuse_summary: [],
+            # opponent_details: [],
```

Comment:
remove?!

Reply:
Dammit - I thought I had got all of them! :)